bundles/org.openhab.binding.ferroamp/README.md

```diff
@@ -0,0 +1,187 @@
+# ferroamp Binding
+
+The ferroamp binding is used to get live data from ferroamp energy systems.
+
+The binding retrieve data from the different parts of the ferroamp energy system such as:
+
+- `Ehub`: EnergyHub Wall and EnergyHub XL.
+- `Sso`: Solar string optimizer.
+- `Eso`: Bidirectional DC/DC converter for connection of battery.
+- `Esm`: Energy Storage Module.
+ 
+
+## Supported Things
+
+This binding is compatible with EnergyHub Wall and EnergyHub XL, and use connection to your local Ehub via LAN or similar.
```

Comment:
I think you have to swap this text (general infomration about the binding) with the text in lines 5-10 that look more specific about the things.